### DISCUSSION TITLE
Saving Fit & Gpx tour files into Mytourbook

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1394] [General]
I did a Windows 11 fresh install and a 24.7.0 Mytourbook reinstall (database deleted!)
I saved  fit and  gpx tour files in other folders.
Can I restore them in MTB, and how?
Thank you

**Comment by FJBDev** [comment]
>  Can I restore them in MTB, and how? Thank you

You can restore your previous database by putting it in [this folder](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/backup)
